TPSWorkflow/__init__.py

```diff
@@ -0,0 +1,129 @@
+import yaml
+import os
+from typing import Dict, List
+
+# define defaults to be used during initialization
+OUT_LABEL = ""
+OUT_FOLDER = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+print(OUT_FOLDER)
+DEFAULT = "<value-here>"
```

Comment:
Is this a temporary placeholder?  When parsing an actual config file, I don't think we want the user to include `<value-here>` anywhere.  That can appear in a template or example file to show what information the user needs to provide.